pkg/constants/constants.go

```diff
@@ -109,6 +109,17 @@ const (
 	DefaultMaxUnhealthy = "100%"
 	// DefaultWorkerMaxUnhealthy is the default maxUnhealthy value for worker node machine health checks.
 	DefaultWorkerMaxUnhealthy = "40%"
+
+	// SignatureAnnotation applied to the bundle during bundle manifest signing.
+	SignatureAnnotation = "anywhere.eks.amazonaws.com/signature"
```

Comment:
should the word "bundle" be in this value? it might help future us to know to which signature this is related.